src/assets/icons/Close.svg

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sorry for not mentioning earlier, but can you add this svg file to the `icons/index` file 😬 thanks!